RCSwitch.h

```diff
@@ -97,14 +104,16 @@ class RCSwitch {
 
     struct Protocol {
         int pulseLength;
+        int startData;
         HighLow syncFactor;
         HighLow zero;
         HighLow one;
+        bool enabled;
```

Comment:
Why is this `enabled` member there? It seems to be unused -- perhaps a left-over from an earlier revision of the PR?